foundry.toml

```diff
@@ -27,8 +27,8 @@ out = 'out-via-ir'
 src = 'src/interfaces/'
 
 [fuzz]
-runs = 1000
-max_test_rejects = 1_000_000
+runs = 800
+max_test_rejects = 2_000_000
```

Comment:
which test requires this?

Reply:
The entire `BPool_Unit_ExitPool` contract, I tried keeping the runs at 1k but would have to increase the `max_rejects` a lot more